### PR TITLE
Setting up Babel/es6/Jasmine

### PR DESCRIPTION
**Pull request by Guntrisoft**
@benbarnett

### CODE REVIEW

**Comment by benbarnett** [comment]
It's pretty self explanatory I guess but maybe just flesh this commit message out a bit? Keeps the history in line with @benlovell's lovely commits!

Otherwise 💯

**Comment by Guntrisoft** [comment]
@benbarnett updated the commit to be more descriptive!

**Comment by benbarnett** [comment]
Nice one 👍

File: lib/tasks/analyse_javascript.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+desc 'Analyse Javascript with JSHint and Javascript Code Style checker'
+task :analyse_javascript do
+  puts 'Running Javascript code analysis...'
+  system('npm run js') || raise('Javascript analysis failed')
```
Comment (benlovell) [inline lib/tasks/analyse_javascript.rake]: Do we need to raise here when we could rely on the exit code of the `npm` sub-process?

Comment (Guntrisoft) [inline lib/tasks/analyse_javascript.rake]: In testing this change locally (removing the `raise`) it seemed to continue to run the Jasmine tests afterwards, so leaving this in for now.

**Pull request merged**